packages/uniforms/__tests__/AutoForm.tsx

```diff
@@ -65,21 +67,21 @@ describe('<AutoForm />', () => {
     });
 
     it('calls `onChangeModel`', () => {
+      const schema = new SimpleSchema({ a: { type: String, optional: true } });
+      const bridge = new SimpleSchema2Bridge(schema);
```

Comment:
Don't we wanna switch to zod also here?

Reply:
Sure, but maybe not in this PR - I'd rather iterate faster on these.